site/ko/blog/new-in-devtools-100/index.md

```diff
@@ -28,19 +28,24 @@ draft: true
 <!-- end: translation instructions -->
```

Comment:
please update the `whats-new.md` file too!

Reply:
Update complete